freqtrade/freqtradebot.py

```diff
@@ -1320,34 +1321,41 @@ def manage_open_orders(self) -> None:
         :return: None
         """
         for trade in Trade.get_open_order_trades():
-            try:
-                if not trade.open_order_id:
+            for open_order in trade.open_orders:
+                try:
+                    order = self.exchange.fetch_order(open_order.order_id, trade.pair)
+
+                except (ExchangeError):
+                    logger.info(
+                        'Cannot query order for %s due to %s', trade, traceback.format_exc()
+                    )
                     continue
-                order = self.exchange.fetch_order(trade.open_order_id, trade.pair)
-            except (ExchangeError):
-                logger.info('Cannot query order for %s due to %s', trade, traceback.format_exc())
-                continue
 
-            fully_cancelled = self.update_trade_state(trade, trade.open_order_id, order)
-            not_closed = order['status'] == 'open' or fully_cancelled
-            order_obj = trade.select_order_by_order_id(trade.open_order_id)
+                fully_cancelled = self.update_trade_state(trade, open_order.order_id, order)
+                not_closed = order['status'] == 'open' or fully_cancelled
+                order_obj = trade.select_order_by_order_id(open_order.order_id)
```

Comment:
I think you can omit this now, and use `open_order` instead.

`open_order` is already the order-object - so this is really irrelevant.

Reply:
Good catch